core/src/main/java/nu/yona/server/subscriptions/service/UserAddService.java

```diff
@@ -89,15 +89,15 @@ private void onStart()
 	public UserDto addUser(UserDto user, Optional<String> overwriteUserConfirmationCode)
 	{
 		assert user.getPrivateData().getDevices().orElse(Collections.emptySet()).size() == 1;
-		UserRetrievalService.assertValidUserFields(user, UserService.UserPurpose.USER);
+		UserLookupService.assertValidUserFields(user, UserService.UserPurpose.USER);
```

Comment:
I would say this `assertValidUserFields` should be part of `UserService` or `UserAddService`, not of `UserLookupService`

Reply:
I wanted to keep the assertions together. As everything depends on the ``UserLookupService``, they ended up there. Alternatively, I could extract another service that has these assertions. What do you think of that?